css/index.css

```
@@ -0,0 +1,101 @@
+body {
+  font-family: sans-serif;
+  background-color: #FFEEDE;
+}
+
+.slide-food-Spin {
+  position: relative;
+  width: 600px;
+  height: 600px;
+  margin: 0 auto;
+  margin-top: 15%;
+}
+
+.carousel-food-Spin {
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+
+.carousel-food-Spin__dishes{
```

El nombre hace pensar que es una lista o un contenedor. Como te estas refiriendo a un elemento `plato`, idealmente hay que escribirlo en singular. Ah, y como te mencione en comentarios anteriores, en minusculas
```suggestion
.carousel-food-spin__dish {
```